src/digisos/skjema/ettersendelse/ettersendelseVedleggListe.tsx

```diff
@@ -1,153 +1,113 @@
-                            >
-                                {tittelKey && (
-                                    <h3>
-                                        <FormattedMessage id={tittelKey} />
-                                    </h3>
-                                )}
-                                {info && <p>{info}</p>}
-                            </EttersendelseVedlegg>
-                        );
-                    })}
+    return (
+        <div
+            className={
+                "ettersendelse__vedlegg__innhold " + (advarselManglerVedlegg ? "ettersendelse__vedlegg__feil " : "")
+            }
+        >
+            {data &&
+                data.map((vedlegg: EttersendelseVedleggBackend) => {
+                    const spc = getSpcForOpplysning(vedlegg.type);
+                    const tittelKey = spc ? spc.textKey + ".vedlegg.sporsmal.tittel" : "";
+                    const infoKey = spc ? spc.textKey + ".vedlegg.sporsmal.info" : "";
+                    let info;
+                    if (infoKey && !!intl.messages[infoKey]) {
+                        info = intl.formatMessage({id: infoKey});
+                    }
+                    if (!props.ettersendelseAktivert && vedlegg.type === "annet|annet") {
+                        return null;
+                    }
+                    return (
+                        <EttersendelseVedlegg
+                            ettersendelseAktivert={props.ettersendelseAktivert}
+                            vedlegg={vedlegg}
+                            key={vedlegg.type}
+                            feilKode={feiletVedleggId === vedlegg.type ? feilKode : undefined}
+                        >
+                            {tittelKey && (
+                                <h3>
+                                    <FormattedMessage id={tittelKey} />
+                                </h3>
+                            )}
+                            {info && <p>{info}</p>}
+                        </EttersendelseVedlegg>
+                    );
+                })}
 
-                <AvsnittMedMarger className="ettersendelse_send_vedlegg_knapp_wrapper">
-                    {this.state.advarselManglerVedlegg && (
-                        <>
-                            <div className="skjema__feilmelding">
-                                <FormattedMessage id="ettersendelse.feilmelding.ingen_vedlegg" />
-                            </div>
-                            <br />
-                        </>
-                    )}
+            <AvsnittMedMarger className="ettersendelse_send_vedlegg_knapp_wrapper">
+                {advarselManglerVedlegg && (
+                    <>
+                        <div className="skjema__feilmelding">
+                            <FormattedMessage id="ettersendelse.feilmelding.ingen_vedlegg" />
+                        </div>
+                        <br />
```

Comment:
pirk: men foretrekker css framfor <br> for space :D

Reply:
Veldig enig. Tar det i neste PR. :)